package.json

```diff
@@ -70,6 +70,16 @@
         "configuration": "./csharp.configuration.json"
       }
     ],
+    "configuration": {
+      "title": "Omnisharp configuration",
```

Comment:
Should this be 'C# configuration' instead since that is the name of this extension?